src/pages/solution/adobe-campaign-classic/tabs/api-reference.md

```diff
@@ -301,14 +303,43 @@ func application(_ application: UIApplication, didReceiveRemoteNotification user
 **Example**
 
 ```objectivec
-- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)launchOptions fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler
+- (void) application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)launchOptions fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler
 {
-  if ( launchOptions) NSLog(@"launchOptions: %@", [launchOptions description]);
-  // Tracking silent push notification receive
-  if ( [launchOptions[@"aps"][@"content-available"] intValue] == 1 ) {
-    NSLog(@"Silent Push Notification");
-    [AEPMobileCampaignClassic trackNotificationReceiveWithUserInfo:userInfo];
-    completionHandler(UIBackgroundFetchResultNoData);
-  }
+    if ( launchOptions) NSLog(@"launchOptions: %@", [launchOptions description]);
+    // Tracking silent push notification receive
+    if ( [launchOptions[@"aps"][@"content-available"] intValue] == 1 ) {
+        NSLog(@"Silent Push Notification");
+        [AEPMobileCampaignClassic trackNotificationReceiveWithUserInfo:userInfo];
+        completionHandler(UIBackgroundFetchResultNoData);
+    }
+}
+```
+
+<Variant platform="android" api="handle-remote-message" repeat="7"/>
+
+_**Requires extension version v2.1.0+**_
+
+Builds an `AEPPushPayload` then constructs a `Notification` using the `RemoteMessage`'s payload. The notification is then passed to the `NotificationManagerCompat` to be displayed. If any exceptions are thrown when building the `AEPPushPayload` or `Notification`, this method will return `false` to signal that the remote message was not handled by the `AEPMessagingService`.
```

Comment:
"Builds an `AEPPushPayload` with data extracted from a `RemoteMessage`'s payload. The built `AEPPushPayload` is then used to construct a `Notification`."